java/typesafe-activator/Portfile

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem       1.0
+
+name             typesafe-activator
+version          1.3.12
+categories       java devel
+platforms        darwin
+supported_archs  noarch
+license          Apache-2
+maintainers      gmail.com:nhoj.patrick \
+                 openmaintainer
+
+description      TypeSafe activator framework used by the java web play framework
+
+long_description \
+    Play Framework makes it easy to build web applications with Java & Scala. \
+    Play is based on a lightweight, stateless, web-friendly architecture. \
+    Built on Akka, Play provides predictable and minimal resource consumption \
+    (CPU, memory, threads) for highly-scalable applications.
+
+homepage         https://www.playframework.com
+
+master_sites     https://downloads.typesafe.com/${name}/${version}/
+distname         ${name}-${version}
+worksrcdir       activator-dist-${version}
+
+checksums        rmd160 22b6dff199f48e7cb59e24b6afd6c8d3a2a3f373 \
+                 sha256 3740355ce4d4c099b9349704dc5851341e731981b25094b7b7dcefe7eceff530
+
+use_configure   no
+use_zip         yes
+
+build           {}
+
+destroot        {
+    set activatordir ${destroot}${prefix}/share/java/${name}
+
+    xinstall -m 755 -d ${activatordir}
+    xinstall -m 755 -d ${activatordir}-bob
```

Comment:
oops debug line i left in as i wanted to triple check what directory ${activatordir} had resolved to.